backend/src/main/java/ubb/courses/backend/repositories/MediaRepository.java

```diff
@@ -0,0 +1,10 @@
+package ubb.courses.backend.repositories;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import ubb.courses.backend.models.Media;
+
+import java.util.List;
+
+public interface MediaRepository extends JpaRepository<Media, Integer> {
+    List<Media> findMediaByCourse_Id(Integer courseId);
```

Comment:
Use set if we do not want to have an implied order because we have orderIndex. Or use list and make sure it is always ordered by the orderIndex.